### PR TITLE
refactor: to work with React@15.5+

### PR DESCRIPTION
**Pull request by benjycui**
ref: https://github.com/ant-design/ant-design/issues/5678

### CODE REVIEW

**Comment by benjycui** [comment]
Updated.

File: tests/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const TestUtils = require('react-addons-test-utils');
```
Comment (afc163) [inline tests/index.js]: react-addons-test-utils => react-dom/test-utils

**Comment by benjycui** [comment]
Updated.

**Comment by benjycui** [comment]
CI failed 是因为 karma 无法启动浏览器，先忽略吧。这个 PR 合并后我改成 Jest

**Comment by benjycui** [comment]
ping~

**Pull request merged**